specification/summary-care-record.yaml

```diff
@@ -30,11 +30,17 @@ paths:
       operationId: operationId
       parameters:
         - in: header
-          name: Nhsd-Asid
+          name: X-Request-ID
```

Comment:
Not yet in this PR but ok, let's leave it as https://gpitbjss.atlassian.net/browse/NIAD-740 will bring this header in